compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -374,7 +374,7 @@ int entry(int argc, char **argv)
   }
   catch (const std::runtime_error &err)
   {
-    std::cout << err.what() << std::endl;
+    std::cerr << err.what() << std::endl;
     std::cout << arser;
```

Comment:
@jinevening, @jyoungyun 
```c++
std::cout << arser;
```
corresponds to help message after error occurred. Should it be printed to cout? or cerr?

If we consider the usage as a part of error message, the second `cout` should be `cerr` also.

Reply:
When I tested in my environment, docker put the additional message to the stderr.

```
$ docker abc
docker: 'abc' is not a docker command.
See 'docker --help'
$ docker abc 2>./abc
# no message
```

refer to https://www.middlewareinventory.com/blog/linux-stdout-stderr-bash/

Reply:
@jyoungyun Thank you for providing `git` and `docker` behaviors. I will update and split this PR.

Reply:
I prefer the latter. The usage message without prefix looks better in the viewpoint of users.

Reply:
I think it would be better if all logs put to stderr, but usage prints without the module name.
But if the implement becomes more complicated, the second example (std::cout << arser) looks better.

Reply:
@jyoungyun Please give your opinion. If you would like to follow `docker` way, I will modify this PR.
But I think it is just difference of preference and my previous way may be more similar to `docker`  in perspective of user.

Here are  further example runs from `docker` and `git`

```
$ docker version 142
"docker version" accepts no arguments.
See 'docker version --help'.

Usage:  docker version [OPTIONS]

Show the Docker version information
```

```
 $ docker run alkdsajflkdsajf
Unable to find image 'alkdsajflkdsajf:latest' locally
docker: Error response from daemon: pull access denied for alkdsajflkdsajf, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```

```
$ git hello world
git: 'hello' is not a git command. See 'git --help'.

The most similar command is
        shell

$ git log akljladksjfasd
fatal: ambiguous argument 'akljladksjfasd': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

They usually don't prepend its name to all lines. Rather, it prepends only for 1st or selected lines only.